### DISCUSSION TITLE
Error: "pdf is not UTF-8 encoded" etc

### DISCUSSION THREAD

**Discussion by PSG2017** [RyotaUshio/obsidian-pdf-plus#200] [Q&A]
Hi, whenever I try to export a pdf in PDF++ it says it's exporting it but it gets stuck in an endless loop. The only way to stop it is to use the Task Manager (in Windows) and ending the Pandoc app. When I do that I get a long error message in Obsidian that ends: "pdf is not UTF-8 encoded: falling back to latin1". I've had no problem exporting Word docs from PDF++. The Pandoc app is installed on my computer, as is PDFLaTeX, and paths have been inserted in the relevant boxes in the Obsidian Pandoc plugin options. Exporting the PDF using the Obsidian Pandoc plugin "Export to PDF" menu item (instead of using the PDF++ "Export to..." menu item) works just fine, except the annotations added in PDF++ do not appear in the exportd pdf - it is just the original unannotated pdf that is exported. Not sure where I am going wrong. Has anyone else experienced this and can you suggest a cure? Thanks in advance. Peter

**Comment by RyotaUshio** [comment]
What do you mean by "export a PDF in PDF++"? PDF++ does NOT offer a function that exports something to PDF, although it can be used to modify pre-existing PDFs. My guess is you are confusing something unrelated to this plugin as related.

**Reply by PSG2017** [reply]
Hi, in the annotated pdf file in Obsidian, click the three dots menu item at top right, in the menu click Export to..., in the popup select PDF. enter Export to address, click Export. If this is not the way to export the annotated pdf, is there any way to open a copy of the annotated pdf in say Abobe Acrobat?

**Reply by RyotaUshio** [reply]
The "Export to..." thing you described is not PDF++'s. I don't know but I guess it may be from the pandoc plugin.

> If this is not the way to export the annotated pdf, is there any way to open a copy of the annotated pdf in say Abobe Acrobat?

- if the annotations are directly written into the PDF file itself, you can open it in Adobe Acrobat to see all the annotations
- If the highlights are "backlink highlighting" (see the PDF++ docs), currently there is no way to export it to actual PDF annotations (actual means being directly written in the file itself and being able to be viewed in Acrobat). It's one of the planned features.

**Reply by PSG2017** [reply]
Thanks very much for that clarification. 

With regard to:

> If the highlights are "backlink highlighting" (see the PDF++ docs), currently there is no way to export it to actual PDF annotations (actual means being directly written in the file itself and being able to be viewed in Acrobat). It's one of the planned features.

If that feature is added then that would make the plugin a true alternative to Zotero (which can export annotated copies of the pdf).

Peter
